cli/init/init.go

```diff
@@ -156,7 +159,13 @@ func generateTtEnv(configPath string, sourceCfg configData) error {
 	}
 	cfg.CliConfig.App.TarantoolctlLayout = sourceCfg.tarantoolctlLayout
 
-	if err := util.WriteYaml(configPath, cfg); err != nil {
+	ttYamlContent, err := util.GetTextTemplatedStr(&ttYamlTemplate, cfg)
+	if err != nil {
+		return err
+	}
+
+	err = os.WriteFile(configPath, []byte(ttYamlContent), 0666)
```

Comment:
Why do we make this file available for writing for non-owner? Other user can change bin_dir to run its own binary instead of the owner's tarantool.

Reply:
Fixed, thank you.